modules/rosa-hcp-sts-creating-a-cluster-ext-auth-provider-cli.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-external-auth-provider-cli_{context}"]
+= Creating an external authentication provider
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+ 
+After you have created a {hcp-title} cluster with the enabled option for external authentication providers, you must create a provider using the ROSA CLI.
+
+[NOTE]
+====
+Similar to the `rosa create|delete|list idp[s]` command in the ROSA CLI, you cannot edit an existing identity provider that you created using `rosa create external-auth-provider`. Instead, you must delete the external authentication provider and create a new one.
+====
+
+This configuration uses Microsoft Entra ID to authenticate your credentials. The Entra ID values are:
```

Comment:
I see the confusion now.

This paragraph is written in a generic / agnostic way, which is good

![image](https://github.com/openshift/openshift-docs/assets/1662329/c82cc3f1-41e7-456b-a0fe-92116e62f1db)

Then I would remove the "Micrososft Entra ID" from here.

I would mention in the "Example output" the provider when it adds specific info (for example, in the "Verification" -> "Example output") but it's minor.